Add documentation for 'subset' and interpolation' arguments (ImageDataGenerator)

StefanoCappellini opened the pull request:
Methods flow (subset) and flow_from_directory (subset + interpolation) are currently missing it.

fchollet commented on keras/preprocessing/image.py:
```diff
@@ -680,6 +680,8 @@ def flow(self, x, y=None, batch_size=32, shuffle=True, seed=None,
                 (only relevant if `save_to_dir` is set).
                 save_format: one of "png", "jpeg" (only relevant if `save_to_dir` is set). Default: "png".
+               subset: Subset of data (`"training"` or `"validation"`) if
+                validation_split is set in ImageDataGenerator.
```
Please put "`" around code keywords.

StefanoCappellini commented:
Hi @fchollet , I updated the style of the doc.
However, I think we should add a guide to the `CONTRIBUTING.md` file because actually the doc style is not uniform:
- Class names are sometimes enclosed in a couple of brackets "`" (https://github.com/StefanoCappellini/keras/blob/138663b428c249b6ba43e3d065a3c71629364112/keras/preprocessing/image.py#L1072) and sometimes not (https://github.com/StefanoCappellini/keras/blob/138663b428c249b6ba43e3d065a3c71629364112/keras/preprocessing/image.py#L1304)
- Possible values: are sometimes enclosed (https://github.com/StefanoCappellini/keras/blob/138663b428c249b6ba43e3d065a3c71629364112/keras/preprocessing/image.py#L58) and sometimes not (https://github.com/StefanoCappellini/keras/blob/138663b428c249b6ba43e3d065a3c71629364112/keras/preprocessing/image.py#L431)
- The same holds for string values

A simple example of this issue is that the lines I added were simply copy and pasted from the ones found in the `DirectoryIterator`class. So now the added lines are not consistent with the old ones.

I would prefer not to change the whole doc style in this commit in order to keep its goal meaningful and compact. Please let me know if you agree.
I could open an issue on this.

fchollet commented:
In general all code keywords should be between backquotes. The only exception would potentially be string values, where it matters less, but even in this case backquotes are still better.
